src/table/selections-utils.js

```diff
@@ -80,3 +80,36 @@ export function selectCell(cell, selState, selDispatch, evt) {
     api.cancel();
   }
 }
+
+export const doSelectCell = (evt, rowValues, rowIndex, colId, selState, selDispatch) => {
```

Comment:
not sure it is the correct way.  If yes, I will refactor it.

Reply:
Is this for selecting multiple?

Reply:
Ah, nevermind. This is a new PR...

Reply:
Yes, 
```
 if (keysPressed.Shift && (evt.key === 'ArrowUp' || evt.key === 'ArrowDown')) {
     doSelectCell(evt, rows, rowIndex, colId, selState, selDispatch);
 }
```